fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/MoreMoreAsyncUtil.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * MoreMoreAsyncUtil.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2019 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.provider.foundationdb;
+
+import com.apple.foundationdb.annotation.API;
+import com.apple.foundationdb.async.AsyncUtil;
+import com.apple.foundationdb.async.MoreAsyncUtil;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.BiFunction;
+
+/**
+ * More helpers in the spirit of {@link AsyncUtil} and {@link com.apple.foundationdb.async.MoreAsyncUtil}.
+ */
+@API(API.Status.UNSTABLE)
+public class MoreMoreAsyncUtil {
```

Comment:
I'm not sure I see the utility in separating this into its own helper function over just passing `fdbDatabase::mapAsyncToSyncException` both places...though I could be wrong. I suspect that this should be marked as `INTERNAL` rather than `UNSTABLE`, at least for now...

Reply:
Removed.